libiocage/lib/Resource.py

```diff
@@ -272,10 +278,10 @@ def getstring(self, key):
             key (string):
                 Name of the jail property to return
         """
-        try:
-            return libiocage.lib.helpers.to_string(self.__getattribute__(key))
-        except AttributeError:
-            return "-"
+        return libiocage.lib.helpers.to_string(
+            self.get(key),
+            none="-"
+        )
```

Comment:
we've seen this exact code already above

Reply:
jail is a resource
why can't it reuse this function?

Reply:
The code here is now re-used instead of being re-implemented.